### PR TITLE
Remove dates from all pages

### PR DESCRIPTION
**Pull request by PalmTreeVI**
Page embedded dates are useless!  These dates are not likely to be updated whenever other texts are updated on the page. It's much better to just use the files modified date.  This simplifies the pages YAML frontmatter which in turn results in less mistakes and no need to know the specific date format.

### CODE REVIEW

**Review by Liquid369 — APPROVED** [review]
utACK 
Only dates modified

**Review by sh-rn — APPROVED** [review]
utACK

LGTM!

File: 02.pivx-core-wallet/04.command-line-core-wallet/article.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'PIVX Command Line Core Wallet'
-date: '14-10-2021 00:00'
+
```
Comment (JSKitty) [inline 02.pivx-core-wallet/04.command-line-core-wallet/article.md:3]: Nit: unnecessary whitespace, but not a 'blocking' issue by any means.

**Review by JSKitty — APPROVED** [review]
**utACK**, only a mere whitespace, also, do we have any documented method of testing changes?
